src/screens/Home.tsx

```diff
@@ -106,21 +28,24 @@ const styles = StyleSheet.create({
 
 export default function Home({ navigation }: RootStackScreenProps<'Home'>) {
   const { navigate } = navigation;
+  const { journals } = useMMKVStorage();
 
   return (
     <SafeAreaView style={styles.container}>
       <Text style={styles.header}>My Journeys</Text>
       <FlatList
-        data={mockJournalEntries}
+        data={journals}
         style={styles.listContainer}
         contentContainerStyle={styles.listContentContainer}
         renderItem={({ item: entry }) => (
           <JournalEntry
             id={entry.id}
-            image={entry.image}
+            image={`${RNFS.DocumentDirectoryPath}/${entry.imagePath
+              .split('/')
+              .at(-1)}`}
```

Comment:
```suggestion
            image={`file://${entry.imagePath}`}
```

This solves the issue. Working fine in both Android and iOS